### ISSUE TITLE
Every time I make a snapcraft application using $ snapcraft it installs same stuff over and over again

### ISSUE THREAD

**Issue by sonuishaq67** [flutter/flutter#63780]
## Steps to Reproduce
- snapcraft init 
- make changes to .yaml file 

1. Run `flutter create bug`.
2. Update the files as follows: ... 
3. ... 

**Expected results:** 

**Actual results:** 
```
Reading package lists... Done
Installing build dependencies: adwaita-icon-theme binutils binutils-common binutils-x86-64-linux-gnu cpp cpp-7 dconf-gsettings-backend dconf-service dpkg-dev fontconfig fontconfig-config fonts-dejavu-core gcc gcc-6-base gcc-7 gcc-7-base gir1.2-atk-1.0 gir1.2-atspi-2.0 gir1.2-freedesktop gir1.2-gdkpixbuf-2.0 gir1.2-gtk-3.0 gir1.2-harfbuzz-0.0 gir1.2-pango-1.0 glib-networking glib-networking-common glib-networking-services gsettings-desktop-schemas gtk-update-icon-cache hicolor-icon-theme humanity-icon-theme icu-devtools libasan3 libasan4 libatk-bridge2.0-0 libatk-bridge2.0-dev libatk1.0-0 libatk1.0-data libatk1.0-dev libatomic1 libatspi2.0-0 libatspi2.0-dev libavahi-client3 libavahi-common-data libavahi-common3 libbinutils libc-dev-bin libc6-dev libcairo-gobject2 libcairo-script-interpreter2 libcairo2 libcairo2-dev libcc1-0 libcilkrts5 libcolord2 libcroco3 libcups2 libdatrie1 libdbus-1-dev libdconf1 libdpkg-perl libdrm-amdgpu1 libdrm-common libdrm-dev libdrm-intel1 libdrm-nouveau2 libdrm-radeon1 libdrm2 libegl-mesa0 libegl1 libegl1-mesa-dev libepoxy-dev libepoxy0 libexpat1-dev libfontconfig1 libfontconfig1-dev libfreetype6-dev libgbm1 libgcc-6-dev libgcc-7-dev libgdk-pixbuf2.0-0 libgdk-pixbuf2.0-common libgdk-pixbuf2.0-dev libgl1 libgl1-mesa-dri libglapi-mesa libgles1 libgles2 libglib2.0-bin libglib2.0-dev libglib2.0-dev-bin libglvnd-core-dev libglvnd-dev libglvnd0 libglx-mesa0 libglx0 libgomp1 libgraphite2-3 libgraphite2-dev libgtk-3-0 libgtk-3-common libgtk-3-dev libharfbuzz-dev libharfbuzz-gobject0 libharfbuzz-icu0 libharfbuzz0b libice-dev libice6 libicu-dev libicu-le-hb-dev libicu-le-hb0 libicu60 libiculx60 libisl19 libitm1 libjbig0 libjpeg-turbo8 libjpeg8 libjson-glib-1.0-0 libjson-glib-1.0-common liblcms2-2 libllvm10 liblsan0 libmpc3 libmpx2 libopengl0 libpango-1.0-0 libpango1.0-dev libpangocairo-1.0-0 libpangoft2-1.0-0 libpangoxft-1.0-0 libpciaccess0 libpcre16-3 libpcre3-dev libpcre32-3 libpcrecpp0v5 libpixman-1-0 libpixman-1-dev libpng-dev libproxy1v5 libpthread-stubs0-dev libquadmath0 librest-0.7-0 librsvg2-2 librsvg2-common libsensors4 libsm-dev libsm6 libsoup-gnome2.4-1 libsoup2.4-1 libstdc++-6-dev libthai-data libthai0 libtiff5 libtsan0 libubsan0 libwayland-bin libwayland-client0 libwayland-cursor0 libwayland-dev libwayland-egl1 libwayland-server0 libx11-6 libx11-data libx11-dev libx11-xcb-dev libx11-xcb1 libxau-dev libxau6 libxcb-dri2-0 libxcb-dri2-0-dev libxcb-dri3-0 libxcb-dri3-dev libxcb-glx0 libxcb-glx0-dev libxcb-present-dev libxcb-present0 libxcb-randr0 libxcb-randr0-dev libxcb-render0 libxcb-render0-dev libxcb-shape0 libxcb-shape0-dev libxcb-shm0 libxcb-shm0-dev libxcb-sync-dev libxcb-sync1 libxcb-xfixes0 libxcb-xfixes0-dev libxcb1 libxcb1-dev libxcomposite-dev libxcomposite1 libxcursor-dev libxcursor1 libxdamage-dev libxdamage1 libxdmcp-dev libxdmcp6 libxext-dev libxext6 libxfixes-dev libxfixes3 libxft-dev libxft2 libxi-dev libxi6 libxinerama-dev libxinerama1 libxkbcommon-dev libxkbcommon0 libxml2 libxrandr-dev libxrandr2 libxrender-dev libxrender1 libxshmfence-dev libxshmfence1 libxtst-dev libxtst6 libxxf86vm-dev libxxf86vm1 linux-libc-dev make mesa-common-dev pkg-config python3-distutils python3-lib2to3 shared-mime-info ubuntu-mono wayland-protocols x11-common x11proto-composite-dev x11proto-core-dev x11proto-damage-dev x11proto-dev x11proto-fixes-dev x11proto-input-dev x11proto-randr-dev x11proto-record-dev x11proto-xext-dev x11proto-xf86vidmode-dev x11proto-xinerama-dev xkb-data xorg-sgml-doctools xtrans-dev zlib1g-dev
Reading package lists... Done
```
and installs flutter again from flutter snap
```
snap "core18" has no updates available
snapd is not logged in, snap install commands will use sudo
Download snap "flutter" (22) from channel "latest/stable"                  
flutter 0+git.142868f from Flutter Team✓ installed 

gtk-common-themes 0.1-36-gc75f853 from Canonical✓ installed
Pulling flutter-extension 
Initializing Flutter
Downloading https://storage.googleapis.com/flutter_infra/releases/stable/linux/flutter_linux_1.20.2-stable.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  528M  100  528M    0     0  15.3M      0  0:00:34  0:00:34 --:--:-- 16.6M
```
<details>
  <summary>Logs</summary>

flutter run -d linux -v
```
[ +110 ms] executing: [/home/toor/flutter/] git -c log.showSignature=false log
-n 1 --pretty=format:%H
[  +30 ms] Exit code 0 from: git -c log.showSignature=false log -n 1
--pretty=format:%H
[        ] 7c6f9dd2396dfe7deb6fd11edc12c10786490083
[        ] executing: [/home/toor/flutter/] git tag --contains HEAD
[ +207 ms] Exit code 0 from: git tag --contains HEAD
[        ] 1.21.0-9.0.pre
[  +13 ms] executing: [/home/toor/flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[   +3 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/dev
[        ] executing: [/home/toor/flutter/] git ls-remote --get-url origin
[   +5 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[  +44 ms] executing: [/home/toor/flutter/] git rev-parse --abbrev-ref HEAD
[   +5 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] dev
[  +52 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +2 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +12 ms] executing: /home/toor/Android/Sdk/platform-tools/adb devices -l
[  +13 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +1 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +62 ms] executing: /home/toor/Android/Sdk/platform-tools/adb devices -l
[  +67 ms] Found plugin hexcolor at /home/toor/flutter/.pub-cache/hosted/pub.dartlang.org/hexcolor-1.0.4/
[  +90 ms] Found plugin hexcolor at /home/toor/flutter/.pub-cache/hosted/pub.dartlang.org/hexcolor-1.0.4/
[  +66 ms] Generating /home/toor/temp/student-db/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java
[  +81 ms] Starting incremental build...
[   +2 ms] Initializing file store
[  +11 ms] Skipping target: gen_localizations
[   +3 ms] complete
[   +5 ms] Launching lib/main.dart on Linux in debug mode...
[   +9 ms] /home/toor/flutter/bin/cache/dart-sdk/bin/dart --disable-dart-dev
/home/toor/flutter/bin/cache/artifacts/engine/linux-x64/frontend_server.dart.snapshot --sdk-root
/home/toor/flutter/bin/cache/artifacts/engine/common/flutter_patched_sdk/ --incremental --target=flutter --debugger-module-names
--experimental-emit-debug-metadata -Ddart.developer.causal_async_stacks=true --output-dill
/tmp/flutter_tools.TZNCDE/flutter_tool.CNVMMV/app.dill --packages .packages -Ddart.vm.profile=false -Ddart.vm.product=false
--bytecode-options=source-positions,local-var-info,debugger-stops,instance-field-initializers,keep-unreachable-code,avoid-closure-call-i
nstructions --enable-asserts --track-widget-creation --filesystem-scheme org-dartlang-root --initialize-from-dill
build/cache.dill.track.dill
[  +19 ms] Building Linux application...
[  +28 ms] <- compile package:StudentProject/main.dart
[  +12 ms] List of devices attached
[   +3 ms] List of devices attached
[  +18 ms] executing: [build/linux/debug/] cmake -G Ninja -DCMAKE_BUILD_TYPE=Debug /home/toor/temp/student-db/linux
[ +441 ms] -- The CXX compiler identification is Clang 10.0.0
[  +15 ms] -- Check for working CXX compiler: /usr/bin/clang++
[ +142 ms] -- Check for working CXX compiler: /usr/bin/clang++ -- works
[   +5 ms] -- Detecting CXX compiler ABI info
[ +135 ms] -- Detecting CXX compiler ABI info - done
[  +12 ms] -- Detecting CXX compile features
[        ] -- Detecting CXX compile features - done
[   +6 ms] -- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1") 
[   +1 ms] -- Checking for module 'gtk+-3.0'
[  +40 ms] --   Found gtk+-3.0, version 3.24.20
[ +115 ms] -- Checking for module 'glib-2.0'
[  +13 ms] --   Found glib-2.0, version 2.64.3
[  +44 ms] -- Checking for module 'gio-2.0'
[  +22 ms] --   Found gio-2.0, version 2.64.3
[  +69 ms] -- Configuring done
[  +11 ms] -- Generating done
[        ] -- Build files have been written to: /home/toor/temp/student-db/build/linux/debug
[   +6 ms] executing: ninja -C build/linux/debug install
[   +5 ms] ninja: Entering directory `build/linux/debug'
[+13468 ms] [1/6] Generating /home/toor/temp/student-db/linux/flutter/ephemeral/libflutter_linux_gtk.so,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_basic_message_channel.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_binary_codec.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_binary_messenger.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_dart_project.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_engine.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_json_message_codec.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_json_method_codec.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_message_codec.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_method_call.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_method_channel.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_method_codec.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_method_response.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_plugin_registrar.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_plugin_registry.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_standard_message_codec.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_standard_method_codec.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_string_codec.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_value.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/fl_view.h,
/home/toor/temp/student-db/linux/flutter/ephemeral/flutter_linux/flutter_linux.h, _phony_
[   +4 ms] [ +129 ms] executing: [/home/toor/flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] [  +32 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] [        ] 7c6f9dd2396dfe7deb6fd11edc12c10786490083
[        ] [        ] executing: [/home/toor/flutter/] git tag --contains HEAD
[        ] [ +186 ms] Exit code 0 from: git tag --contains HEAD
[        ] [        ] 1.21.0-9.0.pre
[        ] [  +19 ms] executing: [/home/toor/flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[        ] [   +5 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] [        ] origin/dev
[        ] [        ] executing: [/home/toor/flutter/] git ls-remote --get-url origin
[        ] [   +4 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] [        ] https://github.com/flutter/flutter.git
[        ] [  +46 ms] executing: [/home/toor/flutter/] git rev-parse --abbrev-ref HEAD
[        ] [   +4 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] [        ] dev
[        ] [  +40 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ] [   +2 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ] [   +7 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ] [        ] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[        ] [        ] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[        ] [ +103 ms] Initializing file store
[        ] [  +15 ms] Skipping target: gen_localizations
[        ] [   +1 ms] unpack_linux: Starting due to {}
[        ] [ +281 ms] unpack_linux: Complete
[        ] [  +21 ms] kernel_snapshot: Starting due to {}
[        ] [  +46 ms] /home/toor/flutter/bin/cache/dart-sdk/bin/dart --disable-dart-dev
/home/toor/flutter/bin/cache/artifacts/engine/linux-x64/frontend_server.dart.snapshot --sdk-root
/home/toor/flutter/bin/cache/artifacts/engine/common/flutter_patched_sdk/ --target=flutter -Ddart.developer.causal_async_stacks=true
-Ddart.vm.profile=false -Ddart.vm.product=false
--bytecode-options=source-positions,local-var-info,debugger-stops,instance-field-initializers,keep-unreachable-code,avoid-closure-call-i
nstructions --enable-asserts --track-widget-creation --packages /home/toor/temp/student-db/.packages --output-dill
/home/toor/temp/student-db/.dart_tool/flutter_build/84eeaa346b13fb5c47325f2ac4c9e96e/app.dill --depfile
/home/toor/temp/student-db/.dart_tool/flutter_build/84eeaa346b13fb5c47325f2ac4c9e96e/kernel_snapshot.d package:StudentProject/main.dart
[   +1 ms] [+10219 ms] kernel_snapshot: Complete
[        ] [ +605 ms] debug_bundle_linux_assets: Starting due to {}
[        ] [ +145 ms] debug_bundle_linux_assets: Complete
[        ] [ +708 ms] Persisting file store
[        ] [   +7 ms] Done persisting file store
[        ] [   +6 ms] build succeeded.
[        ] [   +7 ms] "flutter assemble" took 12,192ms.
[        ] [ +156 ms] ensureAnalyticsSent: 152ms
[        ] [   +8 ms] Running shutdown hooks
[        ] [   +1 ms] Shutdown hooks complete
[        ] [   +1 ms] exiting with code 0
[ +646 ms] [2/6] Building CXX object CMakeFiles/StudentProject.dir/flutter/generated_plugin_registrant.cc.o
[   +5 ms] [3/6] Building CXX object CMakeFiles/StudentProject.dir/main.cc.o
[  +33 ms] [4/6] Building CXX object CMakeFiles/StudentProject.dir/my_application.cc.o
[ +184 ms] [5/6] Linking CXX executable StudentProject
[        ] [5/6] Install the project...
[  +10 ms] -- Install configuration: "Debug"
[        ] -- Installing: /home/toor/temp/student-db/build/linux/debug/bundle/StudentProject
[        ] -- Set runtime path of "/home/toor/temp/student-db/build/linux/debug/bundle/StudentProject" to "$ORIGIN/lib"
[        ] -- Installing: /home/toor/temp/student-db/build/linux/debug/bundle/data/icudtl.dat
[   +1 ms] -- Installing: /home/toor/temp/student-db/build/linux/debug/bundle/lib/libflutter_linux_gtk.so
[ +135 ms] -- Installing: /home/toor/temp/student-db/build/linux/debug/bundle/data/flutter_assets
[        ] -- Installing: /home/toor/temp/student-db/build/linux/debug/bundle/data/flutter_assets/AssetManifest.json
[        ] -- Installing: /home/toor/temp/student-db/build/linux/debug/bundle/data/flutter_assets/FontManifest.json
[        ] -- Installing: /home/toor/temp/student-db/build/linux/debug/bundle/data/flutter_assets/kernel_blob.bin
[  +36 ms] -- Installing: /home/toor/temp/student-db/build/linux/debug/bundle/data/flutter_assets/fonts
[        ] -- Installing: /home/toor/temp/student-db/build/linux/debug/bundle/data/flutter_assets/fonts/MaterialIcons-Regular.otf
[   +1 ms] -- Installing: /home/toor/temp/student-db/build/linux/debug/bundle/data/flutter_assets/NOTICES
[   +1 ms] -- Installing: /home/toor/temp/student-db/build/linux/debug/bundle/data/flutter_assets/packages
[        ] -- Installing: /home/toor/temp/student-db/build/linux/debug/bundle/data/flutter_assets/packages/cupertino_icons
[        ] -- Installing: /home/toor/temp/student-db/build/linux/debug/bundle/data/flutter_assets/packages/cupertino_icons/assets
[        ] -- Installing:
/home/toor/temp/student-db/build/linux/debug/bundle/data/flutter_assets/packages/cupertino_icons/assets/CupertinoIcons.ttf
[  +10 ms] Building Linux application... (completed in 15.7s)
[ +446 ms] Observatory URL on device: http://127.0.0.1:38013/ZFLlOFl84Yc=/
[   +7 ms] Caching compiled dill
[  +39 ms] Connecting to service protocol: http://127.0.0.1:38013/ZFLlOFl84Yc=/
[   +1 ms] DDS is currently disabled due to https://github.com/flutter/flutter/issues/62507
[ +211 ms] Successfully connected to service protocol: http://127.0.0.1:38013/ZFLlOFl84Yc=/
[   +1 ms] Waiting for Linux to report its views...
[   +8 ms] Waiting for Linux to report its views... (completed in 7ms)
[  +11 ms] DevFS: Creating new filesystem on the device (null)
[  +19 ms] DevFS: Created new filesystem on the device (file:///tmp/student-dbNUIDEH/student-db/)
[   +3 ms] Updating assets
[ +155 ms] Syncing files to device Linux...
[   +1 ms] Scanning asset files
[   +3 ms] <- reset
[        ] Compiling dart to kernel with 0 updated files
[   +1 ms] <- recompile package:StudentProject/main.dart 1f29d4c5-7722-48a0-99ad-9e260b386f8d
[        ] <- 1f29d4c5-7722-48a0-99ad-9e260b386f8d
[ +325 ms] Updating files
[ +115 ms] DevFS: Sync finished
[        ] Syncing files to device Linux... (completed in 448ms)
[        ] Synced 1.7MB.
[   +1 ms] <- accept
[   +1 ms] Connected to _flutterView/0xd041c0.
[   +1 ms] Flutter run key commands.
[   +4 ms] r Hot reload. 🔥🔥🔥
[   +2 ms] R Hot restart.
[   +1 ms] h Repeat this help message.
[   +1 ms] d Detach (terminate "flutter run" but leave application running).
[   +1 ms] c Clear the screen
[   +1 ms] q Quit (terminate the application on the device).
[        ] An Observatory debugger and profiler on Linux is available at: http://127.0.0.1:38013/ZFLlOFl84Yc=/


```

```
no issues found
```

```
[✓] Flutter (Channel dev, 1.21.0-9.0.pre, on Linux, locale en_IN)
    • Flutter version 1.21.0-9.0.pre at /home/toor/flutter
    • Framework revision 7c6f9dd239 (5 days ago), 2020-08-09 10:31:03 -0400
    • Engine revision 6d86e67f04
    • Dart version 2.10.0 (build 2.10.0-4.0.dev 0341576448)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.0)
    • Android SDK at /home/toor/Android/Sdk
    • Platform android-30, build-tools 30.0.0
    • Java binary at: /home/toor/Downloads/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[✓] Linux toolchain - develop for Linux desktop
    • clang version 10.0.0-4ubuntu1
    • cmake version 3.16.3
    • ninja version 1.10.0
    • pkg-config version 0.29.1

[✓] Android Studio (version 4.0)
    • Android Studio at /home/toor/Downloads/android-studio
    • Flutter plugin version 48.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.48.0)
    • VS Code at /usr/share/code
    • Flutter extension version 3.13.2

[✓] Connected device (3 available)
    • Linux (desktop)  • linux      • linux-x64      • Linux
    • Web Server (web) • web-server • web-javascript • Flutter Tools
    • Chrome (web)     • chrome     • web-javascript • Google Chrome 84.0.4147.125

• No issues found!

```

</details>

**Comment by sonuishaq67** [comment]
Uses so much disk space of mine 
it was 382.7 GB left before starting and It's 376.2GB now

**Comment by sonuishaq67** [comment]
Sorry closed it by mistake

**Comment by sonuishaq67** [comment]
Just checked the disk analyzer and these two apps were made today each of them takes 15gb why??
![Screenshot from 2020-08-14 22-10-48](https://user-images.githubusercontent.com/55189255/90272647-4c42a080-de7b-11ea-8b00-28d626e257d7.png)

**Comment by stuartmorgan** [comment]
Snapcraft is not part of the Flutter project. You'll need to report this via Snapcraft reporting channels.
